airbyte-integrations/connectors/destination-azure-blob-storage/src/main/java/io/airbyte/integrations/destination/azure_blob_storage/AzureBlobStorageConsumer.java

```diff
@@ -84,39 +94,30 @@ protected void startTracked() throws Exception {
     }
   }
 
-  private boolean createContainers(final AppendBlobClient appendBlobClient,
-                                   final ConfiguredAirbyteStream configuredStream) {
+  private void createContainers(final SpecializedBlobClientBuilder specializedBlobClientBuilder,
+                                final AppendBlobClient appendBlobClient,
+                                final ConfiguredAirbyteStream configuredStream) {
     // create container if absent (aka SQl Schema)
     final BlobContainerClient containerClient = appendBlobClient.getContainerClient();
     if (!containerClient.exists()) {
       containerClient.create();
     }
-    // create a storage container if absent (aka Table is SQL BD)
-    if (SyncMode.FULL_REFRESH.equals(configuredStream.getSyncMode())) {
-      // full refresh sync. Create blob and override if any
+    if (DestinationSyncMode.OVERWRITE.equals(configuredStream.getDestinationSyncMode())) {
       LOGGER.info("Sync mode is selected to OVERRIDE mode. New container will be automatically"
-          + " created or all data would be overridden (if any) for stream:" + configuredStream
+              + " created or all data would be overridden (if any) for stream:" + configuredStream
```

Comment:
[OPTIONAL] As we move to Java 17, I believe here we can to replace String concatenation with Text blocks.

Reply:
I will leave it unchanged